classes/class-p4-search.php

```diff
@@ -793,16 +793,5 @@ public function view_paged_posts() {
 				}
 			}
 		}
-
-		/**
-		 * Load assets only on the search page.
-		 */
-		public function enqueue_public_assets() {
-			if ( is_search() ) {
-				wp_register_script( 'search', get_template_directory_uri() . '/assets/js/search.js', [ 'jquery' ], '0.2.8', true );
-				wp_localize_script( 'search', 'localizations', $this->localizations );
```

Comment:
this breaks some parts in the search, because the global var 'localizations' is not populated
https://github.com/greenpeace/planet4-master-theme/blob/b01d4569122e680f880c70123117198175928fd1/assets/js/search.js#L133

Reply:
Yes, indeed. The reason I included `search.js` to the main bundle is to pipe it through babel. But we miss the 'localizations' variable that way.